recipe/meta.yaml

```diff
@@ -1,35 +1,42 @@
+{% set name = "jupyter_server_ydoc" %}
 {% set version = "0.8.0" %}
+# there is a test dependency on jupyter_server, but jupyter_server has
+# a runtime dependency on this package. Build first with true, then
+# after jupyter_server is available bump build number and rebuild with false.
+{% set break_cycle = true %}
 
 package:
-  name: jupyter_server_ydoc
+  name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/j/jupyter_server_ydoc/jupyter_server_ydoc-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source//{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
   sha256: a6fe125091792d16c962cc3720c950c2b87fcc8c3ecf0c54c84e9a20b814526c
 
 build:
   number: 0
-  noarch: python
-  script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation --ignore-installed --no-cache-dir -vvv
+  skip: true # [py<37 or s390x]
```

Comment:
Could you add a comment about which package is missing on s390x? Is it `ypy-websocket`?